### DISCUSSION TITLE
Feature request: Allow tying session types to dates

### DISCUSSION THREAD

**Discussion by shaib** [pretalx/pretalx#1800] [Ideas]
We have a 2-day event. One day is for talks, one is for workshops.

(this is very raw, needs much more details before it can really become a feature, just putting it here so I don't forget about it later)

**Comment by rixx** [comment]
What would the impact of a day-limited session type be? Schedule warnings if a session type is scheduled on the wrong day?

**Comment by shaib** [comment]
No, half-baked, not needed, not sure what I was thinking when I wrote this. I'll open a new discussion for the one I really need.
